user_management/api/views.py

```diff
@@ -124,7 +124,7 @@ def post(self, request, *args, **kwargs):
 
         email = serializer.data['email']
         try:
-            user = User.objects.get(email__iexact=email)
+            user = User.objects.get_by_natural_key(email)
```

Comment:
Do we have a default implementation of `get_by_natural_key`? Does it preseve the `iexact` lookup?